doc/cascadia/profiles.schema.json

```diff
@@ -684,8 +684,20 @@
         },
         "useTabSwitcher": {
           "default": true,
-          "description": "When set to \"true\", the \"nextTab\" and \"prevTab\" commands will use the tab switcher UI.",
-          "type": "boolean"
+          "description": "When set to \"true\" or \"mru\", the \"nextTab\" and \"prevTab\" commands will use the tab switcher UI, with most-recently-used ordering. When set to \"inOrder\", the tab switcher will switch tabs in their current ordering. Set to \"false\" to disable the tab switcher.",
```

Comment:
hmm is there a way we can _only_ accept the three strings "mru", "inOrder", and "disabled"? imo that's clearer than giving the option of true/false as well

Reply:
It's too late, we already started accepting `true` and `false`, and telling people to use those settings ;___;

So now they've gotta mean something sensible in the enum world as well